frontend/src/components/table/row.tsx

```diff
@@ -46,14 +46,23 @@ export const DashboardTableRow = ({
     rowText,
     pathName,
   }: {
-    rowText: string | number;
+    rowText: string | number | boolean;
     pathName: string;
```

Comment:
how can somethig that is called rowText be a boolean?

Reply:
fixed